src/main/interface/cli.c

```diff
@@ -4464,8 +4466,9 @@ const clicmd_t cmdTable[] = {
     CLI_COMMAND_DEF("flash_write", NULL, "<address> <message>", cliFlashWrite),
 #endif
 #endif
-#ifdef USE_RX_FRSKY_SPI
-    CLI_COMMAND_DEF("frsky_bind", "initiate binding for FrSky SPI RX", NULL, cliFrSkyBind),
+#ifdef USE_RX_BIND
+    CLI_COMMAND_DEF("frsky_bind", "initiate binding for FrSky SPI RX", NULL, cliRxBind),
```

Comment:
I always wanted to change this to be applicable to more than just the FrSky SPI RX, and rename it to `bind` (i.e. make FlySky support it too). I guess the renaming to `bind` should be done now.

Reply:
change to bind